src/main/_docs/use-che-as-an-ide/ide-ssh.md

```diff
@@ -20,7 +20,10 @@ We then provide various client utilities (authored in Docker!) for connecting to
 # List Workspaces  
 You can get a list of workspaces in a Che server that have an SSH agent deployed. These are the workspaces that you can SSH into.
 ```shell  
-$ che action list-workspaces
+$ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
I think on any given page we should show the full syntax one time, and then all other examples on the same markdown page can use the shortened syntax.  That would be a compromise.